c++/src/capnp/compat/http-over-capnp.capnp

```diff
@@ -84,6 +84,12 @@ interface HttpService {
     #
     # Client -> Server WebSocket frames will be sent via method calls on `upSocket`, while
     # Server -> Client will be sent as calls to `downSocket`.
+
+    startOptimizedWebSocket @2 (headers :List(HttpHeader), upStream :ByteStream)
```

Comment:
is this pumping websocket-specific? The signature looks generic enough: could this be generic "pump" call?

Reply:
This is intended to extend the optimizedPumpTo support on KJ-HTTP websockets to http-over-capnp.
Generic http doesn't really have the concept of pump as it's a request-response model.

This specific call is very WS specific as implemented.